assets/app/view/game/map.rb

```diff
@@ -92,9 +92,16 @@ def map_y
       end
 
       def render_map
-        w_size, h_size = @layout == :flat ? [85, 50] : [50, 85]
-        width = (@cols.size * w_size) + GAP
-        height = (@rows.size * h_size) + GAP
+        flat2flat = 85
+        edge_length = 50
```

Comment:
make these constants

i'm not sure what flat2flat means maybe choose a better name and use snake case

Reply:
flat2flat = flat side to flat side = [apothem](https://mathworld.wolfram.com/Apothem.html) * 2